### ISSUE TITLE
Audit the SIG ContribEx GitHub team

### ISSUE THREAD

**Issue by palnabarun** [kubernetes/community#8047]
### Describe the issue

Historically the GitHub team was used to document the contributors working in a specific community group. As of now, it's not very relevant as GitHub team membership is not tied to group membership.

- [ ] Cleanup https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-contributor-experience/teams.yaml

/sig contributor-experience
/priority important-longterm
/kind cleanup

/assign

**Comment by BenTheElder** [comment]
Yeah, but we could do something like asking leads to review this periodically with some low frequency? There has to be a better answer than filing one-off issues.

**Comment by palnabarun** [comment]
Agreed. This should be something that should be part of an yearly-checklist of each lead.

**Comment by palnabarun** [comment]
(partly this is how I created the issue)
